server/plugin/webhook.go

```diff
@@ -88,6 +88,9 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if config.EnableWebhookEventLogging {
```

Comment:
I'm inclined to move this check at the start of the handler to ensure the event get's logged before any checks happen. One motivation of the ticket is to make debugging easier. That is hard, if only a sub group of incoming requests are logged.

Reply:
it is on top after we parse the event